contrib/automation_tests/test_cases/live_tab.py

```diff
@@ -87,3 +87,20 @@ def _execute(self, function_name, min_calls=0, max_calls=pow(2, 33)-1):
         self.expect_true(min_calls <= call_count <= max_calls,
                          'Call count for function "%s" expected to be between %s and %s, was %s'
                          % (function_name, min_calls, max_calls, call_count))
+                         
+class VerifyOneFunctionWasCalled(LiveTabTestCase):
+    """
+    Verify that at least one of the functions matching the function name has 
+    received the given number of hits.
+    """
+    def _execute(self, function_name, min_calls=0, max_calls=pow(2, 33)-1):
+        children = self.find_control('Tree', parent=self._live_tab).children()
+        for i in range(len(children)):
+            if function_name in children[i].window_text():
+                call_count = int(children[i + 1].window_text())
+                if min_calls <= call_count <= max_calls:
```

Comment:
But isn't `min_calls` 0 here?

Reply:
0 is the default value for the parameter. Can be anything.

Reply:
Of course, but you `min_calls` is not passed to `VerifyOneFunctionWasCalled` in `orbit_instrument_libs_.py`, so in that test the value of `min_calls` is 0 and you're not actually verifying that the function was called at least once.

Reply:
I did the same for `VerifyFunctionCallCount` above, but I'd agree with Davide that the default is a bit strange here. I think it comes down to the name of the test... in `VerifyFunctionCallCount`, verifying a count of 0 sounds like a reasonable usecase. In `VerifyOneFunctionWasCalled`, I'd expect a minumum of 1.

But for consistency, do you maybe want to change the default `min_calls` to 1 in both tests?

Reply:
Ah, okay, yes. That's fine but I can also put the ususal range in there for consistency. done.

Reply:
Now I finally got it. Yes, zero is not a great value for the check. I'll make it 1.